.npmignore

```diff
@@ -42,3 +42,5 @@ bower_components/**
 gulpfile.js
 !content/themes/casper/gulpfile.js
 package-lock.json
+config.*.json
+!core/server/config/env/**
```

Comment:
@kirrg001 Adding `!core/server/config/**` instead here will still allow a custom config file - `core/server/config/config.production.json` to get published. Using above allows files inside `/env/*` to still pass through as expected.

Reply:
Yeah that works too. As long as core/server/config files are part of the tgz. But that should be the case, because none of these files use this notation.

Reply:
Ah yeah true. Basically means any config file outside of `core/server/config` will never go in. 👍